src/pages/account/[account]/collection/[collection].tsx

```
@@ -37,10 +79,101 @@
+        route: `assets/${assetId}`,
+      });
+      const uris = response.data.asset.uris;
+      const metadataUri = uris?.find(
+        (uri: { key: string; value: string }) => uri.key === 'metadata',
+      );
+      const nftMetadata = metadataUri ? metadataUri.value : null;
+      setMetadata(nftMetadata);
+    } catch (error) {
+      console.error('Error fetching metadata:', error);
+      throw error;
+      setMetadata(null);
```

Biome (2.1.2)

[error] This code will never be reached ...

... because this statement will throw an exception beforehand


(lint/correctness/noUnreachable)